src/ontology/mondo-edit.obo

```diff
@@ -20095,17 +20095,17 @@ xref: MEDGEN:510600 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: MESH:D008944 {source="DOID:11502", source="MONDO:equivalentTo"}
 xref: NANDO:2200303 {source="MONDO:NANDO", source="https://orcid.org/0000-0003-0011-764X", source="https://orcid.org/0000-0002-0170-9172"}
 xref: NCIT:C50852 {source="DOID:11502"}
-xref: NCIT:C50888 {source="DOID:11502", source="MONDO:equivalentTo"}
```

Comment:
NCIT:C197881 should be added as equivalent x-ref, especially since it is used as an x-ref for the definition.

Reply:
added

Reply:
added